common/playbar.js

```diff
@@ -120,37 +134,27 @@ export function createPlaybar () {
 
     }
 
-    playbarhtml.find("#myRange").on('input', updateSlider);
+    playbarhtml.find("#myRange").on('input', function() {
+      updateSlider();
+    });
```

Comment:
Why not '=> notation'? What standard/style are we using?

Reply:
Why I did it this way is that I am restoring code that I had removed. Really, this wants to be remodularized a bit. I'll do that now, and clean up a little tiny mess.

Reply:
okay sounds good! Ping me when I should look again

Reply:
Changes look much better